### DISCUSSION TITLE
Automatic docstring generation - Python

### DISCUSSION THREAD

**Discussion by mrGoonies** [zed-industries/zed#8355] [Feature Requests]
Is there a way to create docstring to document my code in Zed or something similar to this ![VSCode extension](https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring) ?

**Comment by Moshyfawn** [comment]
There is currently no functionality to generate docstrings in the editor.

There's a Tasks proposal in #7108 that might help you achieve your use-case. They've recently released its initial version in #8009. 

P.S. Your "VScode extension" link is broken 😉
